.ci/generate_bear_requirements.py

```diff
@@ -124,7 +134,7 @@ def _to_entry(requirement, default_operator):
 
 
 def _get_requirements(requirements, default_operator, exclude=[]):
-    return dict(
+    return CommentedMap(
```

Comment:
if you are adding the `CommentedMap` in `sort_requirements`, I think you dont need it here. (not 100% sure as I havent tested this PR yet)